internal/generate/crd/crd.go

```diff
@@ -273,6 +275,7 @@ func (g Generator) generateNonGo() (map[string][]byte, error) {
 
 	sort.Sort(k8sutil.CRDVersions(crd.Spec.Versions))
 	setCRDStorageVersion(crd)
+	setSDKstamps(crd, g.OperatorType)
```

Comment:
```suggestion
	g.setSDKstamps(crd)
```

Reply:
This _could_ be a method, but is more easily unit-tested if it isn't.